Library/Formula/mono.rb

```diff
@@ -3,6 +3,7 @@ class Mono < Formula
   homepage "http://www.mono-project.com/"
   url "http://download.mono-project.com/sources/mono/mono-4.0.2.5.tar.bz2"
   sha256 "b074584eea5bbaaf29362486a69d70abe53d0d2feb334f231fa9c841cf6fd651"
+  revision 1
```

Comment:
I bumped the revision because mono compiled eg. on yosemite will fail on el cap unless this fix is included. That's also the reason the patch is replied regardless of OS X version.